src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs

```diff
@@ -165,6 +167,29 @@ private void CleanExistingPackageReferences(ProjectRootElement outputMSBuildProj
             }
         }
 
+        private void AutoInjectImplicitProjectJsonAssemblyReferences(NuGetFramework framework, List<ProjectLibraryDependency> packageDependencies)
+        {
+            if (framework?.IsDesktop() ?? false)
+            {
+                InjectAssemblyReferenceIfNotPresent("System", packageDependencies);
+                if (framework.Version >= new Version(4, 0))
```

Comment:
@livarcocc I don't think it needs to be. It won't change (will always be there as a prereq for including Microsoft.CSharp). Unless for stylistic reasons you want it? Then where should I define this constant?